pkg/stanza/operator/input/udp/udp.go

```diff
@@ -39,7 +39,8 @@ func NewConfigWithID(operatorID string) *Config {
 	return &Config{
 		InputConfig: helper.NewInputConfig(operatorID, operatorType),
 		BaseConfig: BaseConfig{
-			Encoding: helper.NewEncodingConfig(),
+			Encoding:    helper.NewEncodingConfig(),
+			TokenizeLog: true,
```

Comment:
Typically we try to use default value of false, in part because it avoids double negatives. Maybe call this `OneLogPerPacket` or something similar?